tests/channels/pjsip/sdp_offer_answer/incoming/nominal/single-media-stream/audio/packetization/sipp/uac-ptime-accept.xml

```diff
@@ -53,7 +53,7 @@
       <ereg regexp="a=ptime:30+..*"
             search_in="body" check_it="true" assign_to="2"/>
       <test assign_to="2" variable="2" compare="equal" value=""/>
-      <ereg regexp="a=maxptime:150+..*"
+      <ereg regexp="a=maxptime:[0-9]+..*"
```

Comment:
Why [0-9]+ rather than just picking the multiple of 20 that will be used? Is this necessary? It just feels overly lenient, if it could really be any value at all.

Reply:
> Why [0-9]+ rather than just picking the multiple of 20 that will be used? Is this necessary? It just feels overly lenient, if it could really be any value at all.

I understand that multiple of 20 was a requirement of a specific interconnect. This amount can be any amount greater than the negotiated amount. I didn't think it was fair if this requirement limited me to create another one that limits someone else without having a plausible justification.

>The commit message does not comply with the commit message guidelines.
I really don't know how to do this, can you help me?

Reply:
I agree that this is overly lenient. I believe it should test for the specific value, to ensure that if something in the future changes it by mistake for some reason we are aware of it.

Reply:
Arguing with the use of Google translate can be difficult, leading to misleading conversation tones due to the different line. Please keep in mind that I really want to participate and be as courteous as possible.

I understand that the SDP test must validate your structure.
The structure of the SDP expects a number and it is being done.

At the moment the codec (only a few) that should validate this field as it is a multiple of its size. Other codecs whether or not to display this parameter. I don't know the codec list that should display this parameter and the list in which the test of being multiple must be validated. Also, personally, unfortunately, I don't know how to cross-check this information with the test environment if I had it.

Any other limitation by regex is artificially impressive and creates a limitation under false pretenses.

Since I am raising here the question that a parameter in the future must be parameterizable, I don't think it's fair to generate this limitation.

If it is parameterizable in the future, a wrong value for the protocol to work may just be an application configuration error and not a code error.
If it is a code error when this is done, it must validate its functionality.

Or do you maintain the guideline of restricting what exactly you expect?
And what can we actually do to follow up.

Reply:
The test is validating things as currently implemented, that maxptime is the explicit hardcoded value. If that changes in the future to be configurable or otherwise, then the test would need to be updated and expanded to cover that scenario.